doc/gh-pages/pages/development/ftbufferplugin.md

```diff
@@ -0,0 +1,70 @@
+---
+title: MNE-SCAN FieldTrip Buffer Plugin
+parent: Develop
+nav_order: 7
+---
+
+# MNE-SCAN FieldTrip Buffer Plugin
+
+The plugin is meant to interface with a FieldTrip Buffer already running at a given address. It does not implement the buffer itself, only a client for the buffer, and currently only a client that reads from the buffer.
+
+It works by connecting to a given address where a FieldTrip buffer is running after having set some data parameters. The plugin will then continuously send requests to receive data from the buffer, and upon receiving it will output that data.
+
+## Building and Running:
+
+These steps assume you already have MNE-Scan. If not please start [here](https://mne-cpp.github.io/pages/started/started.html).
+
+1. Download FieldTrip Buffer source code:
+
+   You can get the source code either from their [website](http://www.fieldtriptoolbox.org/download/) or their [github page](https://github.com/fieldtrip/fieldtrip). The website download gives you the option to download only the buffer source code. The github repository has the buffer source code located under `realtime/src/buffer`.
+
+   The plugin was made using buffer source code dated to 2019-12-13, and as the fieldtrip c++ wrappers are still under development, different version may not be compatible.
+
+2. Move the source code to your plugin directory:
+
+   The `src` and `cpp` folders need to be placed in `applications/mne_scan/plugins/ftbuffer/ftsrc`.
+
+3. Include ftbuffer in the build process:
+
+   The lines of code that include the ftbuffer plugin in `applications/mne_scan/libs/scShared/Management/pluginmanager.cpp`, `applications/mne_scan/libs/scShared/scShared.pro`, `applications/mne_scan/mne_scan.pro`, and `applications/mne_scan/plugins/plugins.pro` are commented out by default.
```

Comment:
I could not find anything in mne_scan.pro

Reply:
sorry, should be applications/mne_scan/mne_scan/mne_scan.pro instead, pushed a fixed version